packages/rum-react/README.md

```diff
@@ -4,3 +4,113 @@
 > This package is experimental. Use at your own risk.
 
 This package provides React and React ecosystem integrations for Datadog Browser RUM.
+
+## Installation
+
+```bash
+npm install @datadog/browser-rum @datadog/browser-rum-react
+```
+
+## Usage
+
+### Initialization
+
+To enable the React integration, pass the `reactPlugin` to the `betaPlugins` option of the `datadogRum.init` method:
+
+```javascript
+import { datadogRum } from '@datadog/browser-rum'
+import { reactPlugin } from '@datadog/browser-rum-react'
+
+datadogRum.init({
+  applicationId: ...,
+  clientToken: ...,
+  ...
+  betaPlugins: [reactPlugin()],
+})
+```
+
+### Error Tracking
+
+To track React component rendering errors, use one of the following:
+
+- A basic `ErrorBoundary` component (see [React documentation][1]) that catches errors and reports them to Datadog.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ I usually try to avoid words such as _basic_, _just_, ...  
```suggestion
- An `ErrorBoundary` component (see [React documentation][1]) that catches errors and reports them to Datadog.
```